hummingbot/connector/exchange/bitmart/bitmart_api_order_book_data_source.py

```diff
@@ -222,7 +228,7 @@ async def listen_for_order_book_diffs(self, ev_loop: asyncio.BaseEventLoop, outp
                 for trading_pair in self._trading_pairs:
                     ws_message: WSRequest = WSRequest({
                         "op": "subscribe",
-                        "args": [f"spot/depth400:{convert_to_exchange_trading_pair(trading_pair)}"]
+                        "args": [f"spot/depth50:{convert_to_exchange_trading_pair(trading_pair)}"]
```

Comment:
What's the reason for lowering depth?

Reply:
Its what is available. 

<img width="799" alt="image" src="https://user-images.githubusercontent.com/8539682/171565142-418c6606-9532-4d55-ba4d-4972715e1150.png">

Reply:
Is the channel `spot/depth400` really invalid or is it just not documented? If it is not invalid we should use it anyway, even if it is not documented.

Reply:
I'll tell you what, That was one of the first fixes I put in for errors. Its possible other things made that an error. 
I'll test it out with 400 again, and see what happens.

Reply:
There do not appear to be any adverse effects to the depth400. 
Someone else should test it first however.

Reply:
What depth is it actually receiving? Really the 400?

Reply:
I am not sure. My client asked me to get something for bitmart, fixing the errors was the most logical pathway given the time constraints. At this point this change is back to what it used to be and is not erroring. So this change (from documentation) was not necessary to fix the bugs. 

Hopefully the repair either gets added in, or becomes a good place for the maintainer to continue their development on the connector.